src/bpftrace.h

```diff
@@ -223,7 +236,11 @@ class BPFtrace
                         BpfBytecode &bytecode,
                         void (*trigger)(void));
   void* ksyms_{nullptr};
+  // note: exe_sym_ is used when layout is same for all instances of program
   std::map<std::string, std::pair<int, void *>> exe_sym_; // exe -> (pid, cache)
+  std::map<int, void *> pid_sym_;                         // pid -> cache
+  std::map<std::string, std::map<uint64_t, elf_symbol, std::greater<uint64_t>>>
```

Comment:
I think uintptr_t would be a better type for storing symbols. It's what we use everywhere else at least

Reply:
Makes sense, on 32-bit platforms it will save some memory.